.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Why is this needed?

Reply:
Was added because I tried to update typescript here, yet the eslint rules use an older version (and npm 7 threw errors due to its new peer dep parsing which basically made peerDeps a non-optional-optionalDependencies)

Personally, I use this everywhere (I much prefer npm <7's behavior), but I don't know if it's wanted here too 👀 Some input would be lovely!

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/v4.29.0/package.json#L124
https://cdn.jsdelivr.net/npm/@apify/eslint-config-ts@0.1.2/package.json

Hmm... I removed `legacy-peer-deps` and it installed just fine. Am I missing something?

Reply:
> Personally, I use this everywhere (I much prefer npm <7's behavior), but I don't know if it's wanted here too eyes

I have never used this so I can't say what's better. I guess it's ok as long as it works, but not sure.

Reply:
Oh, right, I was installing `@next` since that was the version that had the fix in, that'd explain why 😅 

Yeah, I can remove this if wanted 👍

Reply:
Let's keep it for now. It's safer. We can make it more strict later.